app/models/loan.rb

```diff
@@ -66,8 +67,12 @@ class Loan < ActiveRecord::Base
   # beware, the methods generated by this include will fail without the corresponding OptionSet records existing in the database
   attr_option_settable :status, :loan_type, :project_type, :public_level
 
-  # results in linked CustomModel instances with the associated CustomFieldSet schema
-  attr_custom_model_linkable :loan_criteria, :loan_post_analysis
+  # provides acess to linked CustomModel instances with the associated CustomFieldSet schema.
+  # note these instances will be 'owned' by the loan's organization and potentially shared by more than one loan
```

Comment:
I thought we had discussed duplicating this data rather than hiding that ownership relationship between the custom instances and organizations? Am I misremembering? The use of `has_one_` here strongly implies that the loan, not the organization, is the owning instance for the data.